doc/visual-programming/source/widgets/visualize/treeviewer.rst

```diff
@@ -1,31 +1,18 @@
 Tree Viewer
 ===========
 
-.. figure:: icons/tree-viewer.png
-
 A visualization of classification and regression trees. 
 
-Signals
--------
-
-**Inputs**:
-
--  **Tree**
-
-   A decision tree.
-
-**Outputs**:
-
--  **Selected Data**
-
-   Data from a selected tree node.
-
--  **Data**
+Inputs
+    Tree
+        decision tree
 
-   Data set with an additional attribute for selection labels.
+Outputs
+    Selected Data
+        instances selected from the tree node
+    Data
+        data with an additional column showing whether a point is selected
```

Comment:
Consider renaming *point* to *instance* in this case. Also in Silhouette plot. (Point is also OK, as in "data point". You decide.)

Reply:
Nope, this was accidentally not fixed when copy-pasting. As said - lots of editing.